webapp/src/blocks.tsx

```diff
@@ -927,17 +965,60 @@ export class Editor extends toolboxeditor.ToolboxEditor {
     }
 
     public moveFocusToFlyout() {
-        // if (this.navigationController) {
-        //     this.navigationController.navigation.focusFlyout(this.editor);
-        // }
+        if (this.keyboardNavigation) {
+            const flyout = this.editor.getFlyout();
+            const element = (flyout as any).svgGroup_ as SVGElement;
```

Comment:
instead of casting to any here, add a method to the Flyout in `pxtblocks/plugins/flyout/cachingFlyout.ts` to return this element and cast to that instead.

Reply:
That makes sense. Done here [4b72884](https://github.com/microsoft/pxt/pull/10512/commits/4b72884fd0be029a4b0e09385faba345a616ddc4)